modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc

```diff
@@ -0,0 +1,86 @@
+[id="installing-image-puller-using-che-operator_{context}"]
+= Installing {image-puller-name-short} using the {prod-short} Operator
+
+This section describes how to use the {prod-short} Operator to install the {image-puller-name-short}. This is a Community-supported technology preview feature.
+
+.Prerequisites
+
+* xref:defining-the-list-of-images-to-pull.adoc[]
+* xref:defining-the-memory-parameters-for-the-image-puller.adoc[]
+ifeval::["{project-context}" == "che"]
+* Operator Lifecycle Manager is available on the {kubernetes} instance.
+endif::[]
+* OperatorHub is available on the OpenShift instance.
```

Comment:
`OperatorHub is available on the OpenShift instance.`

@themr0c this is always true. Do we need this line?

Reply:
What it's not clear is if it's possible to install the image puller on a platform different from OpenShift 4.x. I think the pre-req here should be:
- OpenShift 4.2 or higher (that assumes that OLM and OperatorHub are here)
- Any Kubernetes distribution where OLM and OperatorHub have been installed

Reply:
yeah, this sounds like a valid pre-req